### PR TITLE
Aave V3 Arbitrum: List GMX

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Add payload.
Add script to deploy.
Add tests.
Add diffs.

### CODE REVIEW

File: src/AaveV3_Arb_Listings_20231107/Listings.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+# Motivation
+
+GMX Protocol is the largest DEX offering derivatives and one of the most popular DeFi's today. GMX Token is also one of the highest traded assets on Arbiturm.
+
+The GMX token is the utility, governance, and fee accrual token for the GMX protocol. The GMX value accrual mechanism is unique to the GMX protocol. GMX is a highly staked asset, with approximately 78% of circulating GMX staked earning an APR of [2.94%](https://app.gmx.io/#/earn). This staking yield may generate borrowing demand for GMX, noting the yield is volatile.
+
+Adding the GMX token to Arbitrum Aave v3 support greater asset diversification. Users will be able to deposit GMX as collateral and borrow GMX leading to additional revenue opportunities for Aave.
+
+This publication contains risk paramters provided by Chaos Labs and Gauntlet. The initial forum discussion can be found [here](https://governance.aave.com/t/temp-check-add-gmx-to-arbitrum-v3/12307/8).
```
Comment (brotherlymite) [inline src/AaveV3_Arb_Listings_20231107/Listings.md:19]:
```suggestion
This publication contains risk parameters provided by Chaos Labs and Gauntlet. The initial forum discussion can be found [here](https://governance.aave.com/t/temp-check-add-gmx-to-arbitrum-v3/12307/8).
```

File: src/AaveV3_Arb_Listings_20231107/AaveV3_Arb_Listings_20231107_Test.t.sol [hunk 2]
```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import 'forge-std/Test.sol';
```
Comment (brotherlymite) [inline src/AaveV3_Arb_Listings_20231107/AaveV3_Arb_Listings_20231107_Test.t.sol:5]:
```suggestion
```

**Comment by efecarranza** [comment]
@brotherlymite updated!

**Comment by efecarranza** [comment]
Differential not reached.

**Pull request closed without merging**